app/services/event_generator/composite_generator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class EventGenerator
+  class CompositeGenerator
+    attr_reader :generators
+
+    def initialize(generators)
+      @generators = generators
+    end
+
+    private
+
+      # Send method and args to generators.
+      def method_missing(m, *args)
```

Comment:
Rather than depending on which argument gets passed and using method_missing, maybe we should just put the EventGenerator interface as methods on the composite?